### DISCUSSION TITLE
v1.0

### DISCUSSION THREAD

**Discussion by JanMarvin** [JanMarvin/openxlsx2#771] [Announcements]
Will update release notes later today

## What's Changed
* [get_sheet_names] add escape argument. closes #252 by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/724
* [vignette] Fix dataTable formula by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/727
* Sharp hex by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/728
* fixes for wb_load() from #688 by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/732
* Fix threaded comment id. closes #731 by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/734
* Flexible merge by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/733
* Various wb$sheet_names related fixes and cleanups in wb_functions.R by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/735
* [wb_to_df] fix date1904 conversion by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/737
* guard against empty cc cases. closes #739 by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/740
* [wb_load] add path to object on. closes #741 by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/742
* fix wb_set_header_footer(). closes #747 by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/748
* restore and improve tab_color by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/750
* copy header/footer code by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/751
* Review documentation and examples by @olivroy in https://github.com/JanMarvin/openxlsx2/pull/749
* provide basename2 for to long strings by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/755
* [fmt_txt] escape xml by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/757
* Export `wb_comment()` by @olivroy in https://github.com/JanMarvin/openxlsx2/pull/758
* [docs] update man page for create_hyperlink() by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/759
* Remove wbChartSheet from exports by @olivroy in https://github.com/JanMarvin/openxlsx2/pull/760
* Vignette names by @olivroy in https://github.com/JanMarvin/openxlsx2/pull/762
* Make `wb_get_named_regions()` a wrapper. by @olivroy in https://github.com/JanMarvin/openxlsx2/pull/764
* Pkgdown family by @olivroy in https://github.com/JanMarvin/openxlsx2/pull/763
* Review threaded comment docs by @olivroy in https://github.com/JanMarvin/openxlsx2/pull/765
* internal: use `wb_comment()` in wb_add_thread + deprecations guide by @olivroy in https://github.com/JanMarvin/openxlsx2/pull/766
* Final reordering in pkgdown by @olivroy in https://github.com/JanMarvin/openxlsx2/pull/768
* Remove duplicated docs + remove internal functions docs. by @olivroy in https://github.com/JanMarvin/openxlsx2/pull/769
* Release 1.0 by @JanMarvin in https://github.com/JanMarvin/openxlsx2/pull/743

**Full Changelog**: https://github.com/JanMarvin/openxlsx2/compare/v0.8...v1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/JanMarvin/openxlsx2/releases/tag/v1.0'>v1.0</a>.</em>

**Comment by olivroy** [comment]
Created a blog post to discuss recent changes. https://olivroy.quarto.pub/creating-cool-xlsx-files-with-openxlsx2-10/
https://fosstodon.org/@olivroy/110951123240305891
